### ISSUE TITLE
ILUG-D Code of Coduct: Please review

### ISSUE THREAD

**Issue by rajudev** [ILUGD/Suggestions#2]
# Code of Conduct for Indian Linux User's Group Delhi

ILUG-D, as part of the greater Free Software/Hardware Community, assumes good faith on all those who wish to take part in ILUG-D's activities. However, some experiences at other meetups, conferences etc. have shown us the need to adopt a Code of Conduct in which we state our expectations of all attendees and organizers during any meetups/conferences/events organized/participated by ILUG-D
    This code of conduct applies to all attendees/organizers at meetups/conferences organized by ILUG-D.
    
### ILUG-D Diversity Statement
 Indian Linux User's Group Delhi welcomes and encourages participation by everyone.
 
 No matter how you identify yourself or how others perceive you: we welcome  you. We welcome contributions from everyone as long as they interact  constructively with our community. 
 While much of the work for our project is technical in nature, we value and  encourage contributions from those with expertise in other areas, and welcome  them into our community. 
### Be excellent to each other
 ILUG-D is committed to providing a safe environment for all participants at meetups/conferences/events. All  attendees are expected to treat all people and facilities with respect and  help create a welcoming environment. If you notice behavior that fails to meet  this standard, please speak up and help to keep ILUG-D as respectful as we  expect it to be.  
 ILUG-D is committed to the ideals expressed in our Diversity Statement  (above). We ask all our  organizers, speakers, volunteers, attendees and guests to adopt these principles.  We are a diverse community. Sometimes this means we need to work harder to  ensure we're creating an environment of trust and respect where all who come  to participate feel comfortable and included.  
 We value your participation and appreciate your help in realizing this goal. 
 
### Be respectful
 Respect yourself, and respect others. Be courteous to those around you. If  someone indicates they don't wish to be photographed, respect that wish. If  someone indicates they would like to be left alone, let them be. Our event  venues and online spaces may be shared with members of the public; please be  considerate to all patrons of these locations, even if they are not involved  in the conference.
 
### Be inclusive
 By default, all presentation material should be suitable for people aged 12  and above. 
 If you could reasonably assume that some people may be offended by your talk,  please state so explicitly in the submission notes. This will be taken into  account by the Organizing Team. In case you are unsure if this applies to you,  please contact the organizers. Please  note that you are solely responsible if anything is deemed inappropriate and  you did not contact the organizer's team beforehand. 
 
### Be aware.
 We ask everyone to be aware that we will not tolerate intimidation,  harassment, or any abusive, discriminatory or derogatory behavior by anyone  at any ILUG-D meetup/conference/event or in related online media. 
 Complaints can be made to the organizers by contacting them. All  complaints made to event organizers will remain confidential and be taken  seriously. The complaint will be treated appropriately and with discretion.  Should event organizers or moderators consider it appropriate, measures they  may take can include: 

- the individuals may be told to apologize
- the individuals may be told to stop/modify their behavior appropriately
- the individuals may be warned that enforcement action may be taken if the behavior continues
- the individuals may be asked to immediately leave the venue and/or will be prohibited from continuing to attend any part of the event
- the incident may be reported to the appropriate authorities   

### What does that mean for me?
 All participants, including event attendees and speakers must not engage in  any intimidation, harassment, or abusive or discriminatory behavior. 
 The following is a list of examples of behavior that is deemed highly  inappropriate and will not be tolerated at any of our meetups/conferences/events: 

- offensive verbal or written remarks related to gender, sexual     orientation, disability, physical appearance, body size, race, or     religion;
- sexual or violent images in public spaces (including presentation slides);
- deliberate intimidation;
- stalking or following;
- unwanted photography or recording;
- sustained disruption of talks or other events;
- unwelcome physical contact or other forms of assault;
- unwelcome sexual attention;
- sexist, racist, or other exclusionary jokes;
- unwarranted exclusion from conference or related events based on age, gender, sexual orientation, disability, physical appearance, body size, race, religion;

 We want everyone to have a good time at our events. 
 
Questions?
 If you’re not sure about anything in this Code of Conduct, please  contact the ILUG-D organizers at ilugd@lists.hserus.net. 
 If you wish to report a violation of this Code of Conduct, please contact the organizers. 
 ### Our Promise to You
 
- We will read every complaint and have several people on that alias that      can help investigate and resolve the complaint.
- We will reply, in writing, as soon as possible to acknowledge the concern     and assure that the matter is being investigated.
- Depending on the situation, we will talk to the reporter, the reported, or     both to determine what mediation and/or action is necessary.
- Depending on the outcome of the investigation and mediation, we reserve     the right to expel people not in compliance with our Code of Conduct from     the venue. ILUG-D, the ILUG-D Organizing Committee and the venue in which     meetup/conference/event is being held will not be held responsible for further costs incurred by the dismissal from the conference.

###### Disclaimer
This code of conduct could be reviewed and modifications could be made to it as appropriate.

###### License
Originally derived from https://www.debconf.org/codeofconduct.shtml

**Comment by rajudev** [comment]
Please review the code of conduct document and put in your opinions/suggestions. 
As discussed previously 
@sourabhtk37 @xeon-zolt and more people who I am not being able to tag here.

**Comment by bhanuvrat** [comment]
conference should be replaced with `meetup` or to be more general an `event`

We should also mention that a repeat offender will be excommunicated and banned from this community including the various online platforms: telegram group, mailing lists, twitter etc and others communities that we collaborate with. The information will also percolate to the HR departments of various companies we collaborate with and may result in loss of employment / employment opportunities for the offender.

**Comment by rajudev** [comment]
Finally someone reviewed the CoC. 
I will do suggested changes and will revert.

**Comment by vipulgupta2048** [comment]
LGTM, 
Should we add a clear section of who and how to approach in time of need ?
If someone needs immediate assistance at the time of meetup/conference then these people can be contacted for anything related to the same.
- Also, if we could expand more on behaviour on the venue and how to respect them. There are often cases of littering at the venue and people loitering around in the premises where they are not allowed. Actions can be taken against them, and would also helps in keeping goodwill between meetup group and venue partner.

**Comment by ghost** [comment]
whoami && 2cents:
I am a newbie to this community. i saw a link on matrix to discuss this issue, so here I am.
A lot of the language is not defined. For example:

> If you notice behavior that fails to meet this standard, please speak up and help to keep ILUG-D as respectful as we expect it to be.

Under "being respectful", the above statement sounds very comforting and assuring. But if we pay close attention there is no *standard* as to what respectful is. Calling people by their first names is considered disrespectful widely in India. We should use "आप" then? That is a rhetorical question, and, i don't expect an answer, and what about your friends whom you may use nicknames with?

So, I hope it's clear from the get go. A lot of those statements of the COC are up for debate and subjective interpretations. In conclusion, In such widely varying words/standards, depending on who you may ask, we cannot have harsh punishments for breaking the rules.

TLDR:
1. These rules are fine as is, serves as a reference for what kind of behaviour we can expect.
2. We may improve and define what some of these definitions actually mean
3. Once it's clear exactly what being respectful, being inclusive, being excellent mean. With no doubt or uncertainty then sure, we can have people lose jobs, reputation for breaking the rules.

COC should aspire to be like the open source, creative commons, free software, you can take a thing and tell whether it's open source, cc, or free software, with no debatte over whether it's free software for old people or in this city but not in that city. which sounds absurd to me.

Here is the same document word-for-word, formatted + my comments at the bottom: 
[coc-ascii.txt](https://github.com/ILUGD/Suggestions/files/2752955/coc-ascii.txt)
